atc/db/migration/migrations/1580135204_add_pipelines_last_update.up.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+  ALTER TABLE pipelines
+    ADD COLUMN "last_updated" timestamp with time zone NOT NULL DEFAULT '1970-01-01 00:00:00';
```

Comment:
Definitely this is a good start! I like this change. But I wonder if we can add a new table, say `pipeline_events` instead of just adding a column to table `pipeline`. I have the PR #4673, where I wanted to record pipeline pause and unpause time. If you add such a `pipeline_events` table, then I can reuse it. The new table will bring an ability of recording pipeline's entire lifecycle.

Reply:
I haven't thought through the scale concerns, but I could imagine https://github.com/concourse/rfcs/pull/33 fitting into a similar extension mechanism.

Reply:
Thanks for the feedback!
I think that's an interesting idea, however I'm worried that by implementing an event logging for the pipelines the solution would have to be quite more complex: e.g. this log could not increase indefinitely so some rotation would have to be put in place etc...

This PR also comes from what agreed on #4809.
So, for now, I'd rather restraint increasing the scope of this PR so broadly.